### PR TITLE
Add ZToEE and ZToMuMu input card for RunIISummer20UL production to DYT…

### PR DESCRIPTION
**Pull request by Lvigilante**
Using LHA PDF set name: 325300

### CODE REVIEW

**Comment by agrohsje** [comment]
Hi @Lvigilante. Thanks for the PR. Let me ask: which process are you using. As I found several bugs in other cards I like to quickly check. 
306000 and 323500 should perform very similar in this phase space. We never checked but problems we had happened at high masses. 
So using -d is fine. nevents has nothing to do with the final production. You should be ok with leaving at default.

**Comment by Lvigilante** [comment]
Hi @agrohsje. I would like to ask production for ZToEE M binned samples with powheg-pythia8 for RunIISummer20UL.

**Comment by agrohsje** [comment]
I mean what is the powheg process name:
http://powhegbox.mib.infn.it/

**Comment by Lvigilante** [comment]
Hi @agrohsje . Excuse me, I do not know how to find the powheg process name. Could you please give me an help?

**Comment by agrohsje** [comment]
Which line do you use to produce the gridpack ?

**Comment by Lvigilante** [comment]
python ./run_pwg_condor.py -p f -i My_DYToEE_NNPDF31_13TeV_MllBinned/ZToEE_M-200To400_NNPDF31_13TeV.input -m ZToEE_M200To400_NNPDF31_13TeV -f my_Zee_M-200To400 -q longlunch -n 1000

I think I made some error on the -m option.

**Comment by Lvigilante** [comment]
And, excuse me again, I have one question about that.

How can I understand which powheg process should I use in the -m option?

Because my idea is to use Z since I want to produce ZToEE samples. Is it correct or should I think to something different?

File: bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/ZToEE_M-120To200_NNPDF31_13TeV.input [hunk 1]
```diff
@@ -0,0 +1,38 @@
+! Z production parameter
+vdecaymode 1      !(1:leptonic decay, 2:muonic decay, 3: tauonic decay,...)
+
+numevts NEVENTS
+ih1   1           ! hadron 1 (1 for protons, -1 for antiprotons)
+ih2   1          ! hadron 2 (1 for protons, -1 for antiprotons)
+ndns1 131         ! pdf set for hadron 1 (mlm numbering)
+ndns2 131         ! pdf set for hadron 2 (mlm numbering)
```
Comment (agrohsje) [inline bin/Powheg/production/2017/13TeV/DYToLL_NNPDF31_13TeV_MllBinned/ZToEE_M-120To200_NNPDF31_13TeV.input:8]: can you remove these lines. the point to the internal pdf's which are not used.

**Comment by agrohsje** [comment]
-m Z is the answer to your question. if you remove the ndns settings in all cards i am happy to merge.

**Comment by Lvigilante** [comment]
Hi @agrohsje. I removed the lines as you suggested.
I just have one final question.

After my last commit, I runned the command:

python ./run_pwg_condor.py -p f -i My_DYToEE_NNPDF31_13TeV_MllBinned/ZToEE_M-200To400_NNPDF31_13TeV.input -m Z -f my_Zee_M-200To400 -q longlunch -n 1000

And I got the following gridpack as output:  Z_slc7_amd64_gcc700_CMSSW_10_6_20_my_Zee_M-200To400.tgz
I got also an error file, run_full_my_Zee_M-200To400_0.err (attached here as a .txt file), which contains some error/warning.

Should I take care of those errors/warning or may I go ahead and produce all the gridpacks?
Thanks for your patience.

[run_full_my_Zee_M-200To400_0.err.txt](https://github.com/cms-sw/genproductions/files/7628157/run_full_my_Zee_M-200To400_0.err.txt)

**Comment by agrohsje** [comment]
you can untar the tarball and do 
./runcmsgrid.sh 1000 1234 1 
and get a meaningful cmsgrid_final.lhe
with proper central weight having the right pdf for scale as well as pdf variations?
if yes to all questions, you can ignore.

**Comment by agrohsje** [comment]
if you want you can also provide the gridpack and i can quickly check. pr looks good to me. let me merge. we can still fix in case of problems. but running powheg locally all looked good.

**Comment by Lvigilante** [comment]
Hi @agrohsje. Happy for the merge. I think everything is fine.
Anyway, I shared the gridpack via CERNBox: https://cernbox.cern.ch/index.php/s/zeG7xtq8cSjbdmC
Just as a double check. Thanks again for your help!!!

**Comment by Lvigilante** [comment]
Hi @agrohsje. Sorry if I am asking questions again. I created the gridpacks. According to [1] I should copy them to  
/store/group/phys_generator/cvmfs/gridpacks/slc7_amd64_gcc700/13TeV/powheg/V2/
which do not exists and I am not able to create it. 
Where should I put them?
Sorry and thanks again

[1]: https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideSubgroupMC#1_1_Using_wmLHE_campaigns

**Comment by sihyunjeon** [comment]
Hi you should contact me for that matter (as this would be one of EXO's sample request)

**Comment by agrohsje** [comment]
gridpack looks good to me!

**Pull request merged**